src/js/tabs/osd.js

```diff
@@ -1013,7 +1013,7 @@ OSD.loadDisplayFields = function() {
             default_position: -1,
             draw_order: 390,
             positionable: true,
-            preview: FONT.symbol(SYM.LINK_QUALITY) + '8'
+            preview: FONT.symbol(SYM.LINK_QUALITY) + '4:100'
```

Comment:
This does not seem correct. According to the code the second parameter has a length of two:

https://github.com/betaflight/betaflight/blob/fcef76b82f785167574aed7f1019c016137ce67d/src/main/osd/osd_elements.c#L945-L959

So it will more of the type `4:99`

And this is only with crossfire. With others protocols it only has one digit, the first. I don't know if we have here the info to know if the user has configured crossfire or not because the size of the element is very different from one type to the other.

Reply:
I've been looking at the code and it seems we don't have here the serial protocol selected.

I don't have crossfire. I usually add the Link Quality just side by side with the RSSI element, something like `L9R99` being L and R the symbol representation of Link Preview and RSSI, but now with this preview this is impossible to do. The RSSI is drawn before so the Link Quality preview overwrites it.

I think we have two solutions for this:
- Send the MSP command to get the serial protocol selected and draw the correct preview
- Let the old preview and the users of crossfire will have a not accurate preview.

@mikeller what do you think about this?

Reply:
LQ can currently show 100, and to my way of thinking that is correct (e.g. receiving 100% of the packets).

Not sure the correct way to resolve the issue with other protocol and a single digit.

![Uploading LQ preview update.PNG…]()

Reply:
If we wanted to show the CRSF specific version of this element, we will have to read the selected RX protocol through MSP, and then only show the CRSF version if this is the protocol selected, and what we have now otherwise.
This is a lot of effort for something that is just a placeholder to give users an easy way to set up their OSD, so my hunch is to not go this way.